core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/ContainerQC.java

```diff
@@ -0,0 +1,35 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import uk.ac.bbsrc.tgac.miso.core.data.QC;
+import uk.ac.bbsrc.tgac.miso.core.data.QualityControllable;
+import uk.ac.bbsrc.tgac.miso.core.data.SequencerPartitionContainer;
+
+@Entity
+@Table(name = "ContainerQC")
+public class ContainerQC extends QC {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)